### DISCUSSION TITLE
Added CES Battery Box icon

### DISCUSSION THREAD

**Discussion by PlutoWRC** [slipx06/sunsynk-power-flow-card#307] [Ideas]
Hello. Lovelace is great. Would it be possible to add a picture of the CES Battery Box? I am attaching his picture... 
![listing-box-3f-home-premium](https://github.com/slipx06/sunsynk-power-flow-card/assets/131709473/f0e86eff-48d2-4fd7-afc2-4337ae2a83af)

**Comment by Pho3niX90** [comment]
Do you have mappings for your status sensor?

**Reply by PlutoWRC** [reply]
entities:
  inverter_voltage_154: sensor.oig_2111232079_ac_out_aco_vr
  inverter_power_175: sensor.oig_2111232079_ac_out_aco_p
  inverter_status_59: sensor.oig_2111232079_box_prms_mode
  day_battery_charge_70: sensor.oig_2111232079_batt_bat_apd
  day_battery_discharge_71: sensor.oig_2111232079_batt_bat_and
  battery_soc_184: sensor.oig_2111232079_batt_bat_c
  battery_power_190: sensor.oig_2111232079_batt_batt_comp_p
  day_grid_import_76: sensor.oig_2111232079_ac_in_ac_ad
  day_grid_export_77: sensor.oig_2111232079_ac_in_ac_pd
  grid_ct_power_172: sensor.oig_2111232079_ac_in_aci_wr
  day_load_energy_84: sensor.oig_2111232079_ac_out_en_day
  nonessential_power: sensor.oig_2111232079_boiler_current_cbb_w
  day_pv_energy_108: sensor.oig_2111232079_dc_in_fv_ad
  pv1_power_186: sensor.oig_2111232079_dc_in_fv_p1
  pv2_power_187: sensor.oig_2111232079_dc_in_fv_p2
  pv1_voltage_109: sensor.oig_2111232079_dc_in_fv_v1
  pv1_current_110: sensor.oig_2111232079_dc_in_fv_i1
  pv2_voltage_111: sensor.oig_2111232079_dc_in_fv_v2
  pv2_current_112: sensor.oig_2111232079_dc_in_fv_i2
  pv_total: sensor.oig_2111232079_dc_in_fv_total
  grid_ct_power_total: sensor.oig_2111232079_ac_in_aci_wtotal
  grid_ct_power_L2: sensor.oig_2111232079_ac_in_aci_ws
  grid_ct_power_L3: sensor.oig_2111232079_ac_in_aci_wt
  grid_voltage: sensor.oig_2111232079_ac_in_aci_vr
  inverter_voltage_L2: sensor.oig_2111232079_ac_out_aco_vs
  inverter_voltage_L3: sensor.oig_2111232079_ac_out_aco_vt
  non_essential_load1: sensor.oig_2111232079_boiler_day_w
  radiator_temp_91: sensor.oig_2111232079_box_temp
  energy_cost_buy: sensor.current_spot_electricity_price
  energy_cost_sell: sensor.current_spot_electricity_price
  load_power_L1: sensor.oig_2111232079_ac_out_aco_pr
  load_power_L2: sensor.oig_2111232079_ac_out_aco_ps
  load_power_L3: sensor.oig_2111232079_ac_out_aco_pt
  battery_temp_182: sensor.oig_2111232079_battery_temp
  battery_current_191: sensor.oig_2111232079_battery_current
  battery_voltage_183: sensor.oig_2111232079_battery_volt
  load_frequency_192: sensor.oig_2111232079_ac_in_aci_f

**Comment by slipx06** [comment]
@PlutoWRC this was added in [4.24.0](https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.24.0)

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/3fa809b4-1bc6-400e-a356-0fce2cc00267)

**Reply by PlutoWRC** [reply]
Good job. Thank you very much...
